loom/script/reflection/lsAssembly.cpp

```diff
@@ -116,6 +116,11 @@ void Assembly::bootstrap()
     {
         Type *type = types[i];
 
+        if (type->getMissing())
```

Comment:
Checks like this are all over the place, this is just an additional one since missing types get removed later now.